### PR TITLE
OCPBUGS#23265: Remove extra colon

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.12+

Issue: https://issues.redhat.com/browse/OCPBUGS-23265

Link to docs preview: https://71664--ocpdocs-pr.netlify.app/openshift-enterprise/latest/nodes/index.html#nodes-overview

QE review: Not applicable

Additional information:

### CODE REVIEW

File: nodes/index.adoc [hunk 1]
```diff
@@ -19,13 +19,13 @@ Using the OpenShift CLI (`oc`) or the web console, you can perform the following
 
 The following components of a node are responsible for maintaining the running of pods and providing the Kubernetes runtime environment.
 
-* Container runtime:: The container runtime is responsible for running containers. Kubernetes offers several runtimes such as containerd, cri-o, rktlet, and Docker.
+* Container runtime: The container runtime is responsible for running containers. Kubernetes offers several runtimes such as containerd, cri-o, rktlet, and Docker.
```
Comment (snarayan-redhat) [inline nodes/index.adoc:22]: You could either remove the bullet and retain double colon
OR
Make the component names bold for highlight

something like this
https://docs.openshift.com/container-platform/4.15/installing/index.html

**Review by snarayan-redhat — COMMENTED** [review]
Just one nit. Looks good otherwise.

**Review by JoeAldinger — COMMENTED** [review]
You can't use bold here. You need to [italicize](https://www.ibm.com/docs/en/ibm-style?topic=format-highlighting#:~:text=so%20large%20that%20...-,Words%20that%20are%20emphasized,-Italic).

**Comment by JoeAldinger** [comment]
I'll merge when it finishes building. @xenolinux

**Comment by snarayan-redhat** [comment]
> You can't use bold here. You need to [italicize](https://www.ibm.com/docs/en/ibm-style?topic=format-highlighting#:~:text=so%20large%20that%20...-,Words%20that%20are%20emphasized,-Italic).

@JoeAldinger IMO, as per this [guideline](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#definition-lists) it should be bold, and more so definition list formatting is recommended.

**Comment by JoeAldinger** [comment]
> > You can't use bold here. You need to [italicize](https://www.ibm.com/docs/en/ibm-style?topic=format-highlighting#:~:text=so%20large%20that%20...-,Words%20that%20are%20emphasized,-Italic).
> 
> @JoeAldinger IMO, as per this [guideline](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#definition-lists) it should be bold, and more so definition list formatting is recommended.

@xenolinux if you want a definition list it has to be formatted properly like [this](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#definition-lists:~:text=Apply%20one%20of%20these%20writing%20styles%20to%20the%20definition%20list.%20Be%20consistent%3B%20do%20not%20mix%20these%20styles%20in%20the%20same%20list.)

File: nodes/index.adoc [hunk 2]
```diff
@@ -19,13 +19,13 @@ Using the OpenShift CLI (`oc`) or the web console, you can perform the following
 
 The following components of a node are responsible for maintaining the running of pods and providing the Kubernetes runtime environment.
 
-* Container runtime:: The container runtime is responsible for running containers. Kubernetes offers several runtimes such as containerd, cri-o, rktlet, and Docker.
+* _Container runtime_: The container runtime is responsible for running containers. Kubernetes offers several runtimes such as containerd, cri-o, rktlet, and Docker.
```
[the remark this answers is not in the document]
Comment (JoeAldinger) [reply nodes/index.adoc:22]: I think that will render as you want with the bold. @xenolinux

**Review by JoeAldinger — COMMENTED** [review]
I think I figured out what you were trying to do and what the bug actually was. Take a look and let me know.

**Comment by xenolinux** [comment]
@JoeAldinger Yes made it bold. Thanks

**Pull request merged**